library/alloc/benches/vec.rs

```diff
@@ -658,73 +658,160 @@ fn random_sorted_fill(mut seed: u32, buf: &mut [u32]) {
     buf.sort();
 }
 
-fn bench_vec_dedup_old(b: &mut Bencher, sz: usize) {
+// Measures performance of slice dedup impl.
+// "Old" implementation of Vec::dedup
```

Comment:
I think this is a bit misleading in the context of this PR. It's "old-old" now. Can we put a version on it "from Rust 1.xx" or something like that?